mozilla_bitbar_devicepool/configuration.py

```diff
@@ -46,6 +47,15 @@
 FILESPATH = None
 CONFIG = None
 
+
+class ConfigurationException(Exception):
+    def __init__(self, message):
+        self.message = message
+
```

Comment:
another blank line so we have two blank lines between the class definitions?

Reply:
Done.